src/pyfaf/repos/dnf.py

```diff
@@ -42,42 +42,49 @@ def __init__(self, name, *urls):
         path is passed).
         """
 
-        super(Yum, self).__init__()
+        super(Dnf, self).__init__()
 
-        self.yum_root = None
-        self.load_config_to_self("yum_root", ["yum.root"], "/")
+        self.dnf_root = None
+        self.load_config_to_self("dnf_root", ["dnf.root"], "/")
         self.name = name
         self.urls = urls
-        self.yum_base = yum.YumBase()
-        self.yum_base.doConfigSetup(init_plugins=False, debuglevel=0,
-                                    root=self.yum_root)
-        self.yum_base.conf.cachedir = get_temp_dir("yum")
-        self.yum_base.disablePlugins()
-        self.yum_base.repos.disableRepo("*")
-
+        self.dnf_base = dnf.Base()
+        self.dnf_base.conf.debuglevel=0
```

Comment:
spaces around `=`. In this file there is quite a few lints, try running `pylint-3 --rcfile=pylintrc src/pyfaf/repos/dnf.py`